test/jdk/java/nio/channels/DatagramChannel/Promiscuous.java

```diff
@@ -192,24 +193,13 @@ static void test(ProtocolFamily family,
         }
     }
 
+    private static boolean supportedByPlatform() {
```

Comment:
Do you mind adding a /**/ comment to this method to say that it returns true if platform allows an IPv6 socket join an IPv4 multicast group?